practice-two/src/scripts/services/localStorage.js

```diff
@@ -1,16 +1,21 @@
-// Class definition for the LocalStorage utility.
-class LocalStorage {
-    constructor() {}
-    
-    save(key, data) {
-        localStorage.setItem(key, JSON.stringify(data));
-    }
+const storage = {
+    getTasks: () => {
+        try {
+            const tasksJSON = localStorage.getItem('tasks');
+            return tasksJSON ? JSON.parse(tasksJSON) : [];
+        } catch (error) {
+            console.error('Error while getting tasks:', error);
```

Comment:
For errors or any message please define it as constants. BTW we should have a notification component to show when get any error